### PR TITLE
fix: [CDS-82253]: Switching scope directly without showing scope switch dialog 

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
### Summary

Removing the scope switch dialog when switching scope from Scope selector, the scope switch will happen without any prompt.

#### Screenshots

Before - 

https://github.com/harness/harness-core-ui/assets/62028553/e5380893-83df-4ecc-af7a-2cae7ef40160

After - 

https://github.com/harness/harness-core-ui/assets/62028553/c6211512-d4e3-48f6-a388-42ac3dd8f70b

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Comment by avinashmadhwani02** [comment]
trigger messagecheck

**Comment by sunnykesh-harness** [comment]
can we attach, before and after videos as this is a clear behaviour change.

**Review by sunnykesh-harness — APPROVED** [review]

File: src/modules/45-projects-orgs/components/ScopeSelector/ScopeSelector.tsx [hunk 1]
```diff
@@ -63,7 +62,6 @@ export const ScopeSelector: React.FC<ScopeSelectorProps> = props => {
   const { path } = getRouteParams<ProjectPathProps & { path: string }>(true, activeLink?.url)
   const history = useHistory()
   const { sideNavState } = useLayoutV2()
-  const { showDialog: showPrimaryScopeSwitchDialog } = usePrimaryScopeSwitchDialog({ closeScopeSelector: onClose })
```
[the remark this answers is not in the document]
Comment (avinashmadhwani02) [reply src/modules/45-projects-orgs/components/ScopeSelector/ScopeSelector.tsx:66]: No it is not. I think we can remove the code later once this approach is tested and finalised.

**Comment by avinashmadhwani02** [comment]
> can we attach, before and after videos as this is a clear behaviour change.

Added before after videos

**Review by MondalSayantan — APPROVED** [review]

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**